elastic/kibana
# fallback to default refresh interval values

## Summary

Closes https://github.com/elastic/kibana/issues/126496

Add a verification to the Stack Monitoring timepicker interval setup. Three use cases need to be covered:
- `timepicker:refreshIntervalDefaults` is set to the default `pause: false / value: 0`: in that case we should override to an active 10s interval which is [our current fallback](https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/public/application/hooks/use_monitoring_time.ts#L25-L26). Issue tracking update of that default https://github.com/elastic/kibana/issues/70562
- `timepicker:refreshIntervalDefaults` has been updated by the user: we should apply the interval value regardless of its state
- the interval has been changed with the timepicker component: that update should be sticky to the current user session


### Testing
- timepicker:refreshIntervalDefaults default value
  - go to Stack Management -> Advanced settings
  - ensure `timepicker:refreshIntervalDefaults` is set to the default `{ pause: false, value: 0 }`
  - open Stack Monitoring in a new tab
  - verify the timefilter refresh interval is active and set to 10s
- timepicker:refreshIntervalDefaults updated value
  - go to Stack Management -> Advanced settings
  - change the value for `timepicker:refreshIntervalDefaults`
  - open Stack Monitoring in a new tab
  - verify the timefilter refresh interval is aligned with the new defaults
- User-defined value
  - update the interval in the timepicker
  - navigate to different applications and come back to Stack Monitoring
  - verify the updated interval is still active
